uint8 SUCCESS=0
uint8 SELF_COLLISION=1
uint8 ENVIRONMENT_COLLISION=2
uint8 INVALID_PARAMETERS=4
uint8 PLANNING_SCENE_UPDATE_FAILED=8
uint8 status
